.gitignore

```diff
@@ -1,3 +1,4 @@
-vendor
-composer.lock
-.php_cs.cache
+/vendor/
+/composer.lock
+/.php_cs.cache
+/.phpunit.result.cache
```

Comment:
2 changes here:

- `.phpunit.result.cache` was  a missing entry, this is an artefact of PHPUnit (8+ IIRC)
- add the leading `/` which is kind of a good practice to ensure those files are ignored for the root of this project only